### PR TITLE
refactor: Refactors earn page

### LINKED ISSUE #165: Refactor earn page
# Bounty
Refactor the Earn Page into a Page directed at developers

## Scope

- Add a better introduction, make it appealing for developers to join us
- fix broken Glassfrog integration https://github.com/leapdao/leapdao-website/issues/158
- remove Grant section https://github.com/leapdao/leapdao-website/issues/129
- CTA: subscribe to newsletter (geek section), join slack, join to next kitchen call (interspace)

## Deliverables

- updated code

## Roles
bounty gardener: Clearwood / 40 DAI 
bounty worker: name / share
bounty reviewer: name / share

## Gardener checklist

- [ ] I have attached one of the `size-` labels to this bounty
- [ ] I have submitted this bounty via http://bounty.leapdao.org/viewform
- [ ] I have added this bounty to bounty board here https://github.com/orgs/leapdao/projects/6

Closes https://github.com/leapdao/leapdao-website/issues/129

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #165 

- [x] Add a better introduction, make it appealing for developers to join us
- [x] Remove Resources page
- [ ]  Fix broken Glassfrog integration #158 #168 (For #158 - the roles fetched and filtered are appearing correctly(3 entries), please let me know if I missed anything here.
For #168 - I tested with yarn version `1.12.3` the build seems to be successful, please let me know if I missed anything here)
- [x] Remove Grant section #129

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@Clearwood @johannbarbie please review. [Adding in case auto review request is not generated].

Apologies for the delay in the PR.

**Comment by agbilotia1998** [comment]
Hi @Clearwood, I updated the PR with the latest changes from master.

**Review by Clearwood — CHANGES_REQUESTED** [review]
The CTAs are missing. 
[here](https://hackmd.io/W4yA4jxYSka0VbdKPE-npg) you can have a look at the hackmd detailing our motivations for reworking the content. 
I am missing the drive to draw developers to us. I want everyone visiting this page to be stoked to work for LeapDAO.

File: src/routes/earn.svelte [hunk 1]
```diff
@@ -175,37 +175,15 @@
   <header>
     <Logo />
     <p>
-      Earn with LeapDAO by completing a bounty, taking on a role, or applying
-      for a grant
+      Want to work with an open source community that is focused on a network combining scalability,
+      ease of use and security — all in one?
+      <br>
+      Want to get rewarded for your efforts while contributing to an open source initiative, join us and
+      earn with LeapDAO by completing a bounty, taking on a role, or applying
```
Comment (Clearwood) [inline src/routes/earn.svelte:182]: We don't have a grant program anymore. :)

Comment (agbilotia1998) [reply src/routes/earn.svelte:182]: Okay, I'll remove the grant program.

**Comment by agbilotia1998** [comment]
Hi @Clearwood, I've made changes to the content. Please review and let me know if it requires any other changes. Regarding the CTAs please let me know if anything else is required.

**Comment by agbilotia1998** [comment]
Hi @Clearwood please review this PR and let me know if any further changes are required in this. Thanks :) and sorry for the ping again!

**Comment by Clearwood** [comment]
Sorry for the late reply, I am missing more information on who and what we are as a community at the core. @mwaa in case you have time would also enjoy your feedback.

**Comment by Clearwood** [comment]
As per our bounty terms and missing updates I am going to challenge the bounty and close the Pull Request.

**Comment by agbilotia1998** [comment]
I was waiting for review from @mwaa. Thanks anyways for giving me opportunity to work on this :)

**Pull request closed without merging**